cloud-consumption/src/test/java/com/sequenceiq/consumption/swagger/SwaggerGenerator.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.redbeams.swagger;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Could you please migrate this to JUnit 5?

Reply:
all of them ? we have such est in every service

Reply:
I meant this one only. Ideally all existing JUnit 4 tests should be also migrated when making any changes to them, but here it is fine to only consider this new class.